src/formatDate.js

```diff
@@ -50,7 +50,39 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const fromSeparator = fromFormat.slice(-1);
+  const toSeparator = toFormat.slice(-1);
+  const formatLength = fromFormat.length;
+  const splitedDate = date.split(fromSeparator);
+  const dateObj = {};
```

Comment:
don't use data type names in variable names, let's create more meaningful variable name here